web/src/beta/lib/reearth-ui/components/Button/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { FC, useCallback, useState } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+export type ButtonProps = {
+  appearance?: "primary" | "secondary" | "dangerous" | "simple";
+  disabled?: boolean;
+  size?: "normal" | "small";
+  iconButton?: boolean;
+  icon?: string; // TODO: Icon Name, Use Icon Component
+  title?: string;
+  onClick?: () => void;
+};
+
+export const Button: FC<ButtonProps> = ({
+  appearance = "secondary",
+  icon = "",
+  disabled = false,
+  size = "normal",
+  iconButton = false,
+  title = "",
+  onClick,
+}) => {
+  const [isMouseOver, setIsMouseOver] = useState<boolean>(false);
```

Comment:
I think we should try avoid use react state if it can be implemented by css. In style component you can use something like this:
```
["&:hover"]: {
   color: theme.content.withBackground, // add other conditions like disabled etc.
 },
```